Update azure-pipelines-1.yml for Azure Pipelines

nithyaganeshng opened the pull request.

symorton commented:
Can you delete replace the azure-pipelines.yml file with this new yaml file (azure-pipelines-1.yml), so that we are not running two different pipelines?

nithyaganeshng commented:
> Can you delete replace the azure-pipelines.yml file with this new yaml file (azure-pipelines-1.yml), so that we are not running two different pipelines?

Sure, will do.

nithyaganeshng commented:
> Can you delete replace the azure-pipelines.yml file with this new yaml file (azure-pipelines-1.yml), so that we are not running two different pipelines?

Renamed the file and added lint task.
